tests/bundle/Command/ExportCommandTest.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Tests\Bundle\Personalization\Command;
+
+use EzSystems\EzRecommendationClient\Exception\InvalidArgumentException;
+use EzSystems\EzRecommendationClient\Exception\MissingExportParameterException;
+
+final class ExportCommandTest extends AbstractCommandTestCase
+{
+    protected function setUp(): void
+    {
+        parent::setUp();
+    }
+
+    protected static function getCommandName(): string
+    {
+        return 'ibexa:recommendation:run-export';
+    }
+
+    public function testThrowExceptionWhenInvalidSiteAccessIsPassed(): void
+    {
+        $siteAccess = 'undefined_siteaccess';
+        $this->expectException(InvalidArgumentException::class);
+        $this->expectExceptionMessage(
+            sprintf('SiteAccess %s doesn\'t exists', $siteAccess)
+        );
+        $this->commandTester->execute(
+            [
+                '--siteaccess' => $siteAccess,
+            ]
+        );
+    }
+
+    /**
+     * @param array<array<array<string>|string>> $parameters
```

Comment:
This seems to be wrong. `$parameters` is only ever `array<string, string>`.

Reply:
Fixed: https://github.com/ezsystems/ezrecommendation-client/pull/67/commits/dad3d112e64e54c1b19c400858a1830655185e15